models/recruiters.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const recruiterModel = firestore.collection('recruiters')
+const userModel = firestore.collection('users')
+
+/**
+ * Add the recruiter data
+ *
+ * @param recruiterData { Object }: Recruiter data object to be stored in DB
+ * @param username { String }: Username String to be used to fetch the about user
+ * @return {Promise<{message: string, id: string, recruiterName: string, userName: string, timestamp: string }>}
+ */
+
+const addRecruiterInfo = async (recruiterData, username) => {
+  try {
+    // Add the recruiter data in DB
+    const recruiterInfo = await recruiterModel.add(recruiterData)
+    // Fetch the recruiter from DB
+    const recruiter = await recruiterModel.doc(recruiterInfo.id).get()
+    // Fetch the user from DB
+    const user = await userModel.where('username', '==', username).limit(1).get()
+    let userName
+    if (!user.empty) {
+      user.forEach(doc => {
+        const userFirstName = doc.data().first_name
+        const userLastName = doc.data().last_name
+        const userEmail = doc.data().email
```

Comment:
We could use object destructuring https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#object_destructuring

Also, if you want to rename them while destructuing, you can do that. This will help: https://wesbos.com/destructuring-renaming

`const { first_name: userFirstName, last_name: userLastName, email:userEmail } = doc.data()`

Reply:
Done!